src/domUpdates.js

```diff
@@ -58,8 +61,8 @@ const createGridHTML = allColumns => {
   return htmlCode;
 }
 
-const renderGrid = () => {
-  const gridData = makeRecipeColumnData(recipeData)
+const renderGrid = (data) => {
```

Comment:
thanks for making this dynamic

Reply:
Yes! Will be helpful in future iterations, so I wanted to reuse your logic!